src/controllers/admin/office.ts

```diff
@@ -76,8 +79,94 @@ export const GetAllAdminHandler = async (req: Request, res: Response) => {
       admins: currentAdmins
     });
   } catch (e) {
+    console.log(e)
     return res.status(500).json({
       message: "Failed to retrieve admin details"
     });
   }
 };
+
+export const EditAdminHandler = async (req: Request & { user: { email: string; role: 'Student' | 'Faculty' } }, res: Response) => {
```

Comment:
Remove `& { user: ... }`. We are not making any verifications within the parameters of the handler. We have the validator to do that.

Reply:
Resolved